client/src/modules/display_metadata/display_metadata.js

```diff
@@ -0,0 +1,208 @@
+angular.module('bhima.controllers')
+  .controller('DisplayMetadataController', DisplayMetadataController);
+
+DisplayMetadataController.$inject = [
+  '$state', 'DisplayMetadataService', 'DataCollectorManagementService', 'NotifyService',
+  'uiGridConstants', 'ModalService', 'PatientService', 'util', 'AppCache', 'SurveyFormService', 'ReceiptModal',
+  'ChoicesListManagementService',
+];
+
+/**
+ * DISPLAY METADATA CONTROLLER
+ * This module is responsible for handling the CRUD operation on DISPLAY METADATA CONTROLLER
+ */
+
+function DisplayMetadataController($state, DisplayMetadata, DataCollectorManagement,
+  Notify, uiGridConstants, ModalService, Patients, util, AppCache, SurveyForm, Receipts,
+  ChoicesList) {
+  const vm = this;
+  vm.gridApi = {};
+  vm.filterEnabled = false;
+  vm.toggleFilter = toggleFilter;
+  vm.format = util.formatDate;
+  vm.search = search;
+  vm.collector = {};
+  vm.changes = {};
+  vm.onRemove = onRemove;
+  vm.displayData = displayData;
+  vm.download = DisplayMetadata.download;
+
+  const cache = new AppCache('display_metadata');
+
+  ChoicesList.read()
+    .then(choicesLists => {
+      vm.choicesLists = choicesLists;
+    })
+    .catch(Notify.handleError);
+
+  if ($state.params.id && $state.params.patient) {
+    // Prevent a non-patient form from being used to collect non-patient data
+    vm.patientData = true;
+  } else {
+    vm.patientData = false;
+
+    if (vm.collectorId) {
+      loadGrid();
+    }
+  }
+
+  // options for the UI grid
+  vm.gridOptions = {
+    appScopeProvider  : vm,
+    enableColumnMenus : false,
+    fastWatch         : true,
+    flatEntityAccess  : true,
+    enableSorting     : true,
+    showColumnFooter  : true,
```

Comment:
Do you do anything with the column footer?  I don't see it being used in my tests.

Reply:
Thanks, I add the number of rows 
![image](https://user-images.githubusercontent.com/8035588/64964678-6f438980-d893-11e9-9188-c935d12ca031.png)